wasmtime-api/tests/examples.rs

```diff
@@ -0,0 +1,38 @@
+use std::env;
+use std::process::{Command, Stdio};
+
+fn run_example(name: &'static str) {
+    let cargo = env::var("CARGO").unwrap_or("cargo".to_string());
+    let pkg_dir = env::var("CARGO_MANIFEST_DIR").unwrap_or(".".to_string());
+    debug_assert!(
```

Comment:
I don't think it technically matters since I believe that debug assertions are enabled in `cargo test --release`, but I think we should use a regular assertion here, just to make the code less surprising to read.

```suggestion
    assert!(
```